rfcs/text/0000-image-base-transcriptomics.md

```diff
@@ -0,0 +1,497 @@
+| HCA Metadata Draft                                    	| EBI                      	| Complete        	| Z. Perova     	|
+| Test Datasets Available                               	| starfish                 	| Complete        	| A. Carr       	|
+| Submission Datasets Available                         	| starfish                 	| Q2 2019         	| A. Carr       	|
+| Analysis Pipelines                                    	| Broad                    	| Q2 2019         	| ?             	|
+| Analysis & Adapter Pipelines                          	| Broad                    	| Q2 2019         	| A. Carr       	|
+| Lira subscription                                     	| Broad                    	| TBD             	| S. Ehsan      	|
+| Data Analysis Service starfish Pipeline Tests         	| Broad                    	| Q2 2019         	| G. Grant      	|
+| Specification for input format                        	| starfish                 	| Complete        	| J. Moore      	|
+| Specification for output format                       	| starfish                 	| Q1 2019         	| A. Carr       	|
+| Input validation tool                                 	| starfish, OME            	| Q3 2019         	| J. Moore      	|
+| Output validation tool                                	| starfish, OME            	| Q2 2019         	| A. Carr       	|
+| Ingest support for the nested arrays of channels      	| EBI                      	| ?               	| ?             	|
+| [Ability to describe files that belong together][a]   	| ?                        	| ?               	| ?             	|
+| [Upload and download of subdirectories][b]             	| ?                        	| ?               	| ?             	|
+| DSS spatial data bundle (or bundless) specification   	| UCSC                     	| ?               	| B. Hannafious 	|
+| DSS scale testing                                     	| UCSC / DCP               	| TBD             	| B. Hannafious 	|
+| Data Browser facets for spatial data                  	| UCSC                     	| TBD             	| H. Schmidt    	|
+| Matrix service integration                            	| starfish                 	| TBD             	| M. Kinsella   	|
+| Upload Bundle for 1M+ files                           	| ?                        	| ?               	| P. Shah       	|
+| Image file download latency testing                   	| ?                        	| ?               	| ?             	|
+
+### **Unresolved Questions**
+
+  - Who are the intended beta users for imaging support and what
+    functionality is being targeted?
+
+  - Are there other roadblocks for imaging support in the DCP?
+
+  - What, if any, additional automation may be needed in the DCP to
+    support imaging?
+
+  - What is the appropriate separation of metadata in the SpaceTx
+    Format and the HCA metadata schema?
```

Comment:
As a high level description, HCA contains metadata about biological entities and protocols used to make the imaged specimen. HCA also has very limited file metadata (name, format, type). HCA will also contain target and probe information to support search. This is may be repeated in the SpaceTx codebook and probes.sam which introduces a validation requirement to ensure continuity. SpaceTx metadata contains more granular fields related to the image. These fields are typically also captured by OME formats. Critically metadata is linked via SpaceTx's experiment.json explicitly (imaged_specimen -> experiment.json -> the rest of the data).

Reply:
Ok. Sounds to me like this question can safely be removed. Thanks Matt.